tests/tests/test_monitor_client.py

```diff
@@ -201,6 +204,9 @@ def prepare_dbus_monitoring(
         shutil.rmtree(tmpdir)
 
 
+@pytest.mark.skipif(
+    isK8S(), reason="not testable in a staging or production environment"
+)
 class TestMonitorClientEnterprise:
```

Comment:
I understand we de-scoped this ones because we cannot get the e-mails. Too bad because then we won't have any testing in staging with a real mender-monitor client.

Reply:
I have an idea about emails, and will follow up with creating a Jira ticket to capture it